{"uuid":"be8a0090-12b2-4200-93b8-0b7bfd859069","historyId":"11a8d0357ea69c6f210b1e402fd98cdf","fullName":"org.example.SeleniumDemoTest.testReadingFromCsvFile","labels":[{"name":"package","value":"org.example.SeleniumDemoTest"},{"name":"testClass","value":"org.example.SeleniumDemoTest"},{"name":"testMethod","value":"testReadingFromCsvFile"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"test-automation-demo"},{"name":"subSuite","value":"org.example.SeleniumDemoTest"},{"name":"host","value":"Mahmoud-Omran"},{"name":"thread","value":"25892@Mahmoud-Omran.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testReadingFromCsvFile","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3","trace":"java.lang.RuntimeException: java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:58)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:159)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:809)\r\n\tat org.testng.internal.Parameters.handleParameters(Parameters.java:734)\r\n\tat org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)\r\n\tat org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:946)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\nCaused by: java.lang.ArrayIndexOutOfBoundsException: Index 3 out of bounds for length 3\r\n\tat baseTests.BaseTest.getLoginTestData(BaseTest.java:74)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:54)\r\n\t... 24 more\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1713616349326,"stop":1713616349328}